package.json

```diff
@@ -85,19 +84,16 @@
     "superagent": "3.8.3"
   },
   "dependencies": {
-    "@artsy/backbone-mixins": "3.0.0",
     "@artsy/cohesion": "4.57.0",
     "@artsy/commerce_helpers": "artsy/commerce_helpers",
+    "@artsy/detect-responsive-traits": "^0.1.0",
```

Comment:
There were a bunch of transient dependencies getting pulled from Reaction that I had to add. Added the same versions used in Reaction.